src/java/us/kbase/templates/module_test_perl_client.vm.properties

```diff
@@ -1,28 +1,27 @@
 use strict;
 use warnings;
+use feature qw( say );
 
-use Data::Dumper;
-use Test::More;
+use Test::Most;
 use Config::Simple;
 use Time::HiRes qw(time);
 use Bio::KBase::AuthToken;
 use installed_clients::WorkspaceClient;
 #if ($example)
 use installed_clients::AssemblyUtilClient;
 #end
-use ${module_name}::${module_name}Impl;
+
```

Comment:
Generally cleaned up this file, converted the tests to use more standard perl test methods, got rid of all the `evals`, and added some more appropriate sample tests to encourage good testing practices.